app/serializers/collection-submission.ts

```diff
@@ -11,16 +11,18 @@ export default class CollectionSubmissionSerializer extends OsfSerializer {
      */
     serialize(snapshot: DS.Snapshot, options: {}) {
         const serialized = super.serialize(snapshot, options);
-        const { data, data: { attributes, relationships } } = serialized;
+        const { data, data: { relationships } } = serialized;
+        data.attributes = data.attributes || { };
```

Comment:
For future reference, in the case where the provider has no collection-specific metadata (no `issue`, `volume`, studyDesign`, etc), the `collection-submission` has no attributes, only relationships. So in this case, `data.attributes` is `undefined`, so we need to instantiate it as an empty object